core/backend/src/IModelDb.ts

```diff
@@ -3196,7 +3199,10 @@ export class StandaloneDb extends BriefcaseDb {
       if (iTwinId !== Guid.empty) // a "standalone" iModel means it is not associated with an iTwin
         throw new IModelError(IModelStatus.WrongIModel, `${filePath} is not a Standalone iModel. iTwinId=${iTwinId}`);
       assert(undefined !== file.key);
-      return new StandaloneDb({ nativeDb, key: file.key, openMode, briefcaseId: BriefcaseIdValue.Unassigned });
+      const db = new StandaloneDb({ nativeDb, key: file.key, openMode, briefcaseId: BriefcaseIdValue.Unassigned });
```

Comment:
I don't think this is appropriate. Standalone iModels can still have channels. It's OK to do it for createEmpty because we know there's nothing there, but not when we open an existing file.

Reply:
Makes sense now. Thanks!